app/uk/gov/hmrc/individualsmatchingapi/controllers/v2/PrivilegedIndividualsController.scala

```diff
@@ -63,7 +63,7 @@ class PrivilegedIndividualsController @Inject()(
           Ok(response)
         }
       }
-    } recover recoveryWithAudit(maybeCorrelationId(request), request.body.toString, s"/individuals/matching/$matchId")
+    } recover recoveryWithAudit(maybeCorrelationId(request), auditHelper.redactBody(Json.parse(request.body.toString)), s"/individuals/matching/$matchId")
```

Comment:
Why are you converting JSON to string and then parsing it again as json?

Reply:
the difference in the two controllers led me to take the side of JsValue as the function type in audit helper. So, to pass the individual controllers request body into that function I had to do that. it needs JS value, request.body here is of AnyContent type 🤦‍♂️ 😅